src/App.jsx

```diff
@@ -1,98 +1,14 @@
 /* eslint-disable max-len */
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+
+import { MovieList } from './components/MovieList/MovieList';
```

Comment:
The imported component 'MovieList' is inconsistent with the task description. It should be named 'MoviesList'.